.github/workflows/_build-and-test-locally.yml

```
@@ -257,7 +257,10 @@ jobs:
           ${cov_prefix} cargo nextest run $CARGO_FLAGS $CARGO_FEATURES -E 'package(remote_storage)' -E 'test(test_real_azure)'
 
       - name: Install postgres binaries
-        run: cp -a pg_install /tmp/neon/pg_install
+        run: |
+          mkdir /tmp/neon/pg_install
+          # Use tar to copy files matching the pattern, preserving the paths in the destionation
+          tar cv pg_install/v* pg_install/build/*/src/test/regress/*.so | tar  xv -C /tmp/neon/pg_install
```

This copied artifacts into `pg_install/pg_install`, should it be something like this:

```suggestion
          mkdir /tmp/neon
          # Use tar to copy files matching the pattern, preserving the paths in the destionation
          tar cv pg_install/v* pg_install/build/*/src/test/regress/*.so | tar  xv -C /tmp/neon
```